test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1304,7 +1304,7 @@ class test_ucp_wireup_fallback_amo : public test_ucp_wireup {
     bool use_device_amo(ucp_ep_h ep) {
         ucp_ep_config_t *ep_config = ucp_ep_config(ep);
 
-        for (ucp_lane_index_t lane = 0; lane < UCP_MAX_LANES; ++lane) {
+        for (ucp_lane_index_t lane = 0; lane < UCP_MAX_FAST_PATH_LANES; ++lane) {
```

Comment:
why not UCP_MAX_LANES?

Reply:
because we reduced amo_lanes array size to UCP_MAX_FAST_PATH_LANES

Reply:
ucs_static_array_size